pkgs/development/tools/vulkan-validation-layers/default.nix

```diff
@@ -51,7 +51,7 @@ stdenv.mkDerivation rec {
     owner = "KhronosGroup";
     repo = "Vulkan-ValidationLayers";
     rev = "v${version}";
-    hash = "sha256-hh/lCXSKq8xmygVsFFOGu79DvBvBPcc1l1e5wQskK7M=";
+    hash = "sha256-YsFhSgL/2YPq0Cbs9rFWzUf17UCfwrqLnhx7AmNNR1M=";
```

Comment:
```
""}}}' from '<brace-enclosed initializer list>' to 'const std::unordered_multimap<unsigned int, RequiredSpirvInfo>'
  310 | };
      | ^
      | |
      | <brace-enclosed initializer list>
/build/source/layers/vulkan/generated/spirv_validation_helper.cpp: In function 'const char* string_SpvCapability(uint32_t)':
/build/source/layers/vulkan/generated/spirv_validation_helper.cpp:692:20: error: 'CapabilityCooperativeMatrixKHR' is not a member of 'spv'; did you mean 'CapabilityCooperativeMatrixNV'?
  692 |          case spv::CapabilityCooperativeMatrixKHR:
      |                    ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
      |                    CapabilityCooperativeMatrixNV
make[2]: *** [layers/CMakeFiles/vvl.dir/build.make:818: layers/CMakeFiles/vvl.dir/vulkan/generated/spirv_validation_helper.cpp.o] Error 1
```

Reply:
I see, SPIRV-Headers version is hardcoded in vulkan-validation-layers/default.nix, let me see how can I fix it.

Reply:
I fixed that dependency, but it also depends in a custom glsang (not stable) built with a custom (spirv-tools) also not stable :cry:

Reply:
Ended up upgrading the other libs by hand.

Reply:
Maybe it would be easier to downgrade the validation layers? Not sure if we want to use the tmp- tags.

Reply:
Is there something else I can do to make this progress?

Reply:
give it a day or four, I am not that much into vulkan to know if the update to the tmp tags is good or bad.

Reply:
The tag in question does not seem to exist.

Reply:
> The tag in question does not seem to exist.

They don't tag, they make branches every time instead of creating a tag, but the branches are frozen in time (is like they don't like using git tags :face_exhaling: )

Reply:
To be clear, I didn't come up with those tags by myself, those are the references they use in a json included in the json-validation-layers project.

Reply:
> The tag in question does not seem to exist.

Gosh you are right, they removed the branches, and it's basically impossible to have (cleanly) updated versions of all the components as some of them depend on specific unreleased commits. 
Originally I got the commits (and from them the branches from this file https://github.com/KhronosGroup/Vulkan-ValidationLayers/blob/v1.3.257/scripts/known_good.json